src/bgw/scheduler.c

```diff
@@ -548,10 +553,18 @@ cmp_next_start(const void *left, const void *right)
 static void
 start_scheduled_jobs(register_background_worker_callback_type bgw_register)
 {
+	List *ordered_scheduled_jobs;
 	ListCell *lc;
 	Assert(CurrentMemoryContext == scratch_mctx);
+
 	/* Order jobs by increasing next_start */
-	List *ordered_scheduled_jobs = list_qsort(scheduled_jobs, cmp_next_start);
+#if PG13_LT
+	ordered_scheduled_jobs = list_qsort(scheduled_jobs, cmp_next_start);
+#else
+	/* PG13 does in-place sort */
+	ordered_scheduled_jobs = scheduled_jobs;
+	list_sort(ordered_scheduled_jobs, cmp_next_start);
```

Comment:
This will modify `scheduled_jobs`, right? Should the PG13_LT version to replace `scheduled_jobs` with the result of sorting, so both code paths are semantically equivalent? (then it can be wrapped into compat function with the same semantics)

Reply:
> Should the PG13_LT version to replace `scheduled_jobs` with the result of sorting

Dumb suggestion from me.

But copying the list before sorting by `list_sort` in PG13 should work.